python/lsst/analysis/ap/nb_utils.py

```diff
@@ -273,3 +275,82 @@ def compare_sources(butler1, butler2, query1, query2,
+    """Convert ra/dec coordinates in an astropy table/pandas data frame to
+    pixel x/y positions.
+    """
+    ra = table['coord_ra']
+    dec = table['coord_dec']
+    x, y = wcs.skyToPixelArray(ra, dec, degrees=degrees)
+    return astropy.table.Table.from_pandas(pd.DataFrame({'x': x, 'y': y}))
+
+
+def display_images(butler, visit, detector, backend="firefly"):
+    """Display the science/template/difference images for a given
+    visit+detector, with sources and mask planes overlaid.
+
+    Parameters
+    ----------
+    butler : `lsst.daf.butler.Butler`
+        Butler to load data from.
+    visit, detector : `int`
+        Visit and detector ids to load data for.
+    backend : str, optional
+        afw display backend to display to (typically "firefly" or "ds9").
+
+    Notes
+    -----
+    There are some unused variables in here that could be made useable with
+    boolean kwargs to define what is being displayed.
+    """
+    diffim = butler.get("difference_image", visit=visit, detector=detector)
+    science = butler.get("preliminary_visit_image", visit=visit, detector=detector)
+    template = butler.get("template_detector", visit=visit, detector=detector)
+    images = {"difference": diffim, "science": science, "template": template}
+    # red
+    unfiltered = butler.get("dia_source_unfiltered", visit=visit, detector=detector)
+    rejected = butler.get("rejected_dia_source", visit=visit, detector=detector)
+    trailed = butler.get("long_trailed_source_detector", visit=visit, detector=detector)
+    # yellow
+    candidate = butler.get("dia_source_unstandardized", visit=visit, detector=detector)
+    standardized = butler.get("dia_source_detector", visit=visit, detector=detector)  # noqa
+
+    dia_source = butler.get("dia_source_apdb", visit=visit, detector=detector)
+    good = dia_source['reliability'] > 0.1
+    # blue
+    good_source = dia_source[good]
+    # red
+    bad_source = dia_source[~good]
+    print(f"{len(unfiltered)} unfiltered")
+    print(f"{len(trailed)} trailed")
+    print(f"{len(candidate)} candidate")
+    print(f"{len(bad_source)} low reliability diaSources")
+    print(f"{len(good_source)} good diaSources")
+
+    marginal = butler.get("marginal_new_dia_source", visit=visit, detector=detector)  # noqa
+    ss_source_detector = butler.get("ss_source_detector", visit=visit, detector=detector)  # noqa
+    sky_source = unfiltered["sky_source"]
+
+    rejected = get_xy_from_source_table(rejected, diffim.wcs)
+    candidate = get_xy_from_source_table(candidate, diffim.wcs)
+    unfiltered = get_xy_from_source_table(unfiltered[~sky_source], diffim.wcs)
+    trailed = get_xy_from_source_table(trailed, diffim.wcs)
+    for frame, label in enumerate(("science", "template", "difference")):
+        display = lsst.afw.display.Display(backend=backend, frame=frame)
+        display.image(images[label], title=label)
+        with lsst.afw.display.Buffering():
```

Comment:
I got an `AttributeError: module 'lsst.afw.display' has no attribute 'Buffering'` when running this as is, but changing this line to the suggested code worked for me.
```suggestion
        with display.Buffering():
```